packages/p2p/src/components/orders/order-details/order-details-chatbox.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Channel, SendBirdProvider } from 'sendbird-uikit';
+import 'sendbird-uikit/dist/index.css';
+
+const OrderDetailsChatbox = ({ token, app_id, user_id, channel_url, nickname }) => {
+    React.useEffect(() => {
+        const interval_header = setInterval(() => {
+            const chat_title_element = document.querySelector('.sendbird-chat-header__title');
+            const chat_avatar = document.querySelector('.sendbird-chat-header__avatar');
+            const acronym = nickname.match(/\b(\w)/g).join('');
+            if (chat_title_element.innerText === 'No title') return;
+            if (chat_title_element.innerText === nickname && chat_avatar.innerText === acronym)
+                clearInterval(interval_header);
+
+            chat_title_element.innerText = nickname;
+            chat_avatar.innerText = acronym;
```

Comment:
just curious, there isn't any API to modify the avatar & Title?

Reply:
ya there isn't the chatUI kit is not really customizable, we will try to email them later to help us provide the props for this. bcause this solution is as you guess right, not really maintainable